pkg/dashboard/resource/apigateway.go

```diff
@@ -62,11 +62,13 @@ func (agr *apiGatewayResource) GetAll(request *http.Request) (map[string]restful
 
 	exportFunction := agr.GetURLParamBoolOrDefault(request, restful.ParamExport, false)
 	projectName := request.Header.Get(headers.ProjectName)
+	functionName := request.Header.Get(headers.FunctionName)
```

Comment:
Why header and not query param?

Reply:
@TomerShor because we pass project name that way, so I decided to do the same with function name, but I don't mind both ways anyway

Reply:
yeah we do the same in `GET /functions`.. not sure why we use headers but ok, let's keep it this way for consistency